addons/kubevela-io/resources/deployment/kubevela-io-deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
```

Comment:
Why not use `webservice` component?

Reply:
There are several reasons.
- Users may need the local docs  first to install Vela.
- When there is problem with the Vela, the doc may not work well.(That's not a good news.)

Reply:
- If users need the doc first, they can use "kubectl apply" to install Vela  with the resources.(And then they can install Vela according the doc.)
- If Vela is something wrong, user can use "kubectl apply" to reinstall Vela with the resources.

Reply:
Then there is a concern that whether Vela will affect already deployed applications.

Reply:
It's fine for the initial version, and this is also a good example about pure K8s object to be as addons.

Reply:
@barnett The problem for the resource versions will be a big concern in Vela. Here is an issue, https://github.com/oam-dev/kubevela/issues/3624#issue-1199716391